src/tcgwars/logic/groovy/TcgStatics.groovy

```diff
@@ -1146,6 +1146,72 @@ class TcgStatics {
   static boolean isGxPerformed(){
     bg.em().retrieveObject("gx_"+my.owner)
   }
+
+  /* General checks for attacks and abilities */
+
+  //  * assertBench
+  //    - Can be used with no arguments, or with a condition and additional text included.
+  //    - If a closure is given, it'll check for any "it" element in bench returning true.
+  //    - Optional params:
+  //      + type: if set, restricts to benched Pokémon of a single specific type.
+  //      + isEX/isGX/isV: Can be expanded if needed. All of these unset will have the method search for any Pokémon no matter what, but if even a single one is set true it'll only filter those that are set true as well.
+  //      + opp: If true, checks for the opponent's bench instead of "my" bench.
+  //      + repText: If true, instead of adding cText at the end of the assert it'll be the only thing printed.
```

Comment:
I don't get the purpose of `repText`, can you rewrite the explanation it in a better way?

Reply:
It allows for a more specific assert fail message, just as a possibility. Should cover way too specific filters better (checking it, it used to have more use before adding the "type" param, since then it'd have said "valid Pokémon that" +"are [N] type." instead of "valid [N] Pokémon"). No immediate use right now but doesn't hurt to keep :P

Reply:
So it allows to override the assertion fail message completely or partially?

Reply:
Ok just checked the logic, it allows override. But I would do it differently. 

First of all, I don't get the value of a partially overriding `cText`. 
It should be either a) all auto-generated b) overriden manually. 

Auto-generation can worked on to produce better strings with multiple conditions as well, instead of "follow the stated condition(s)" clause. 

If auto-generated message is worked out to provide a good message, there would be no need for a manual override. 

If you think, manual override is still needed, the field name should be `message`. When set, it should override no matter what, so no need for `cText` and `repText` fields, IMO.

Reply:
> So it allows to override the assertion fail message completely or partially?

Yeah, that'd be the gist of it. Updated the description a bit, wasn't very clear looking at it again.

Reply:
Taking a bit to reply, enabled Github's beta redesign and comments don't autoupdate as usual >.>

Reply:
I quite liked the design change! (but not sure about the push updates yet)